--- a/browser/base/content/navigator-toolbox.inc.xhtml	2022-11-30 09:17:31.000000000 +0000
+++ b/browser/base/content/navigator-toolbox.inc.xhtml	2022-11-30 09:17:46.083853599 +0000
@@ -22,12 +22,14 @@
 # shared with other top level windows in macWindow.inc.xhtml.
 #include browser-menubar.inc
       </toolbaritem>
-      <spacer flex="1" skipintoolbarset="true" style="-moz-box-ordinal-group: 1000;"/>
 #include titlebar-items.inc.xhtml
     </toolbar>
 
+  </vbox>
+
     <toolbar id="TabsToolbar"
-             class="browser-toolbar titlebar-color"
+             class="browser-toolbar"
+             style="-moz-box-ordinal-group: 1000;"
              fullscreentoolbar="true"
              customizable="true"
              customizationtarget="TabsToolbar-customization-target"
@@ -36,8 +38,6 @@
              context="toolbar-context-menu"
              flex="1">
 
-      <hbox class="titlebar-spacer" type="pre-tabs"/>
-
       <hbox flex="1" align="end" class="toolbar-items">
         <toolbartabstop/>
         <hbox id="TabsToolbar-customization-target" flex="1">
@@ -100,20 +100,14 @@
         </hbox>
       </hbox>
 
-      <hbox class="titlebar-spacer" type="post-tabs"/>
-
       <hbox class="private-browsing-indicator"/>
       <hbox id="private-browsing-indicator-with-label">
         <image class="private-browsing-indicator-icon"/>
         <label data-l10n-id="private-browsing-indicator-label"></label>
       </hbox>
 
-#include titlebar-items.inc.xhtml
-
     </toolbar>
 
-  </vbox>
-
   <toolbar id="nav-bar"
            class="browser-toolbar"
            data-l10n-id="navbar-accessible"
