--- a/browser/base/content/browser.xhtml	2020-10-17 07:25:00.102974343 +0000
+++ b/browser/base/content/browser.xhtml	2020-10-17 07:53:06.877051595 +0000
@@ -737,12 +737,13 @@
 # shared with other top level windows in macWindow.inc.xhtml.
 #include browser-menubar.inc
         </toolbaritem>
-        <spacer flex="1" skipintoolbarset="true" style="-moz-box-ordinal-group: 1000;"/>
 #include titlebar-items.inc.xhtml
       </toolbar>
+    </vbox>
 
       <toolbar id="TabsToolbar"
-               class="browser-toolbar titlebar-color"
+               class="browser-toolbar"
+               style="-moz-box-ordinal-group: 1000;"
                fullscreentoolbar="true"
                customizable="true"
                customizationtarget="TabsToolbar-customization-target"
@@ -798,29 +799,11 @@
           </hbox>
         </hbox>
 
-        <hbox class="titlebar-spacer" type="post-tabs"/>
-
-#ifndef XP_MACOSX
         <button class="accessibility-indicator" tooltiptext="&accessibilityIndicator.tooltip;"
                 aria-live="polite"/>
         <hbox class="private-browsing-indicator"/>
-#endif
-
-#include titlebar-items.inc.xhtml
-
-#ifdef XP_MACOSX
-        <!-- OS X does not natively support RTL for its titlebar items, so we prevent this secondary
-             buttonbox from reversing order in RTL by forcing an LTR direction. -->
-        <hbox id="titlebar-secondary-buttonbox" dir="ltr">
-          <button class="accessibility-indicator" tooltiptext="&accessibilityIndicator.tooltip;" aria-live="polite"/>
-          <hbox class="private-browsing-indicator"/>
-          <hbox id="titlebar-fullscreen-button"/>
-        </hbox>
-#endif
       </toolbar>
 
-    </vbox>
-
     <toolbar id="nav-bar"
              class="browser-toolbar"
              aria-label="&navbar.accessibleLabel;"
