--- a/browser/base/content/navigator-toolbox.inc.xhtml	2021-11-29 14:01:10.000000000 +0000
+++ b/browser/base/content/navigator-toolbox.inc.xhtml	2021-12-11 20:48:24.878581570 +0000
@@ -24,12 +24,14 @@
 # shared with other top level windows in macWindow.inc.xhtml.
 #include browser-menubar.inc
       </toolbaritem>
-      <spacer flex="1" skipintoolbarset="true" style="-moz-box-ordinal-group: 1000;"/>
 #include titlebar-items.inc.xhtml
     </toolbar>
 
+  </vbox>
+
     <toolbar id="TabsToolbar"
-             class="browser-toolbar titlebar-color"
+             class="browser-toolbar"
+             style="-moz-box-ordinal-group: 1000;"
              fullscreentoolbar="true"
              customizable="true"
              customizationtarget="TabsToolbar-customization-target"
@@ -38,8 +40,6 @@
              context="toolbar-context-menu"
              flex="1">
 
-      <hbox class="titlebar-spacer" type="pre-tabs"/>
-
       <hbox flex="1" align="end" class="toolbar-items">
         <hbox id="TabsToolbar-customization-target" flex="1">
           <tabs id="tabbrowser-tabs"
@@ -83,18 +83,12 @@
         </hbox>
       </hbox>
 
-      <hbox class="titlebar-spacer" type="post-tabs"/>
-
       <button class="accessibility-indicator" data-l10n-id="navbar-accessibility-indicator"
               aria-live="polite"/>
       <hbox class="private-browsing-indicator"/>
 
-#include titlebar-items.inc.xhtml
-
     </toolbar>
 
-  </vbox>
-
   <toolbar id="nav-bar"
            class="browser-toolbar"
            data-l10n-id="navbar-accessible"
